scripts/api/data/licenseError.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think just adding it to the json dir in /src/test/resources with a descriptive name should be okay

Reply:
@qqmyers on second thought the rest of the tests also use the scripts/api/data/ directory for the tests, it would make more sense to change that as a whole then for all Integration Tests? Changing the directory to /src/test/resources just for the license test wouldn't make a lot of sense I think

Reply:
@scolapasta , @pdurbin - any guidance for where test data should go? It seems like existing test data is not all in one place, so I'm not sure what the recommendation would be for new test data.

Reply:
So I think putting all the test data there makes sense too - (not sure if I commented on all the test files, but I wasn't trying to single this one out.)